tests/test_utils.py

```diff
@@ -1337,6 +1338,65 @@ def safe_remove(path):
             print(e)
 
 
+def create_files_for_regex_partitioner(
+    root_directory_path: str, directory_paths: list = None, test_file_names: list = None
+):
+    if not directory_paths:
+        return
+
+    if not test_file_names:
+        test_file_names: list = [
+            "alex_20200809_1000.csv",
+            "eugene_20200809_1500.csv",
+            "james_20200811_1009.csv",
+            "abe_20200809_1040.csv",
+            "will_20200809_1002.csv",
+            "james_20200713_1567.csv",
+            "eugene_20201129_1900.csv",
+            "will_20200810_1001.csv",
+            "james_20200810_1003.csv",
+            "alex_20200819_1300.csv",
+        ]
+
+    base_directories = []
+    for dir_path in directory_paths:
+        if dir_path is None:
+            base_directories.append(dir_path)
+        else:
+            data_dir_path = os.path.join(root_directory_path, dir_path)
+            os.makedirs(data_dir_path, exist_ok=True)
+            base_dir = str(data_dir_path)
+            # Put test files into the directories.
+            for file_name in test_file_names:
+                file_path = os.path.join(base_dir, file_name)
+                with open(file_path, "w") as fp:
+                    fp.writelines([f'The name of this file is: "{file_path}".\n'])
+            base_directories.append(base_dir)
+
+
+def create_files_in_directory(
+    directory: str, file_name_list: List[str], file_content_fn=lambda: "x,y\n1,2\n2,3"
+):
+    subdirectories = []
+    for file_name in file_name_list:
+        splits = file_name.split("/")
+        for i in range(1, len(splits)):
+            subdirectories.append(os.path.join(*splits[:i]))
+    subdirectories = set(subdirectories)
+
+    for subdirectory in subdirectories:
+        os.makedirs(os.path.join(directory, subdirectory), exist_ok=True)
+
+    for file_name in file_name_list:
+        file_path = os.path.join(directory, file_name)
+        with open(file_path, "w") as f_:
+            f_.write(file_content_fn())
+
+
+def create_fake_data_frame():
+    return pd.DataFrame({"x": range(10), "y": list("ABCDEFGHIJ"),})
+
+
```

Comment:
Should these functions be in this PR?

Reply:
Nope! These are now gone after a merge from develop